src/omniscapeTransformer.py

```
@@ -549,22 +593,44 @@
 
-    # Build command with threading based on calculated julia_workers_per_tile
+    # Build command - parallelization is controlled via config.ini
+    # We still set -t flag to ensure Julia has threads available for Omniscape to use
     if julia_workers_per_tile > 1:
         runOmniscape = f"{jlExe} -t {julia_workers_per_tile} {runFile}"
-        ps.environment.update_run_log(f"Julia threading: {julia_workers_per_tile} threads")
+        ps.environment.update_run_log(f"Julia threading ENABLED with {julia_workers_per_tile} threads")
     else:
         runOmniscape = f"{jlExe} {runFile}"
-        ps.environment.update_run_log(f"Julia threading: disabled (single-threaded)")
+        ps.environment.update_run_log(f"Julia threading DISABLED (single-threaded)")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)